applications/GeoMechanicsApplication/tests/test_pressure_line_element/test_oblique_line_element2D3N/MaterialParameters.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": [{
+    "model_part_name"                      : "PorousDomain.filter",
+    "properties_id"                        : 1,
+    "Material" : { 
+      "constitutive_law" : {
+        "name"                             : "GeoLinearElasticPlaneStrain2DLaw"
+      },
+      "Variables" : {
+        "IGNORE_UNDRAINED"                 : false,
+        "YOUNG_MODULUS"                    : 1.000000e+07,
+        "POISSON_RATIO"                    : 0.000000e+00,
+        "DENSITY_SOLID"                    : 2.650000e+03,
+        "DENSITY_WATER"                    : 1.000000e+03,
+        "POROSITY"                         : 1.000000e-01,
+        "BULK_MODULUS_SOLID"               : 9.000000e+19,
+        "BULK_MODULUS_FLUID"               : 1.000000e+20,
+        "PERMEABILITY_XX"                  : 9.084000e-06,
+        "DYNAMIC_VISCOSITY"                : 1.000000e-03,
+        "THICKNESS"                        : 1.000000e+00,
+        "BIOT_COEFFICIENT"                 : 1.000000e+00,
+        "RETENTION_LAW"                    : "SaturatedLaw",
+        "SATURATED_SATURATION"             : 1.000000e+00,
+        "RESIDUAL_SATURATION"              : 0.000000e+00,
+        "VAN_GENUCHTEN_AIR_ENTRY_PRESSURE" : 0.000000e+00,
+        "VAN_GENUCHTEN_GN"                 : 0.000000e+00,
+        "VAN_GENUCHTEN_GL"                 : 0.000000e+00,
+        "MINIMUM_RELATIVE_PERMEABILITY"    : 0.000000e+00,
+        "TRANSVERSAL_PERMEABILITY"         : 9.084000e-06
```

Comment:
Unused material parameters? Mostly for unsaturated behaviour, TRANSVERSAL_PERMEABILITY probably is for an interface element material